omics/transfer/download.py

```diff
@@ -3,7 +3,7 @@
 import socket
 from typing import IO, Any, List, Union
 
-from botocore.exceptions import IncompleteReadError, ReadTimeoutError
+from botocore.exceptions import IncompleteReadError, ReadTimeoutError  # type: ignore
```

Comment:
You should be able to get type annotations for these exceptions from [botocore-stubs](https://pypi.org/project/botocore-stubs/).

Just run: `poetry add botocore-stubs` and you should be able to remove the type ignore.

Reply:
Fixed